src/components/sgc-menu-item/sgc-menu-item.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Component,
+  h,
+  Prop,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+} from '@stencil/core';
+import styles from './sgc-menu-item.css';
+
+@Component({
+  tag: 'sgc-menu-item',
+  shadow: true,
+  styles,
+})
+export class SgcMenuItem {
+  @Prop({ reflect: true, attribute: 'empty' }) empty: boolean = false;
+  @Prop({ reflect: true, attribute: 'active' }) active: boolean = false;
+  @Prop({ reflect: true, attribute: 'child' }) child: boolean = false;
+  @Prop({ reflect: true, attribute: 'reviewed' })
+  reviewed: boolean | string = false;
```

Comment:
We are currently still in the process of defining naming conventions for this library, but so far, we have always opted to use prefixes for our boolean variables (e.g. `is`, `has`, `can`, `should`). I would add this here too, at least for the sake of consistency.

```suggestion
  @Prop({ reflect: true, attribute: 'empty' }) isEmpty: boolean = false;
  @Prop({ reflect: true, attribute: 'active' }) isActive: boolean = false;
  @Prop({ reflect: true, attribute: 'child' }) isChild: boolean = false;
  @Prop({ reflect: true, attribute: 'reviewed' })
  isReviewed: boolean | string = false;
```
The attribute names can be left as-is, which means your CSS and example page does not need to be adjusted.